xrmocap/utils/eval_utils.py

```diff
@@ -137,6 +161,27 @@ def align_convention_mask(pred_keypoints3d_raw: Keypoints,
     else:
         gt_keypoints3d = gt_keypoints3d_raw
 
+    pred_kps3d_mask = pred_keypoints3d.get_mask()
+    pred_kps3d = pred_keypoints3d.get_keypoints()[..., :3]
+    if pred_nose is not None:
+        pred_kps3d = add_campus_jaw_headtop(pred_nose, pred_kps3d)
```

Comment:
If `pred_kps3d` 's convention is not campus, we cannot use `add_campus_jaw_headtop`. Shall we add `eval_kps3d_convention == campus`